deploy/clowdapp.yaml

```diff
@@ -357,7 +357,7 @@ parameters:
   name: SERVICE_LOG__URL
 # cloudwatch
 - name: LOGGING_TO_CLOUD_WATCH_ENABLED
-  value: "true"
+  value: "false"
   required: true
 - name: CLOUDWATCH_DEBUG
   value: "true"
```

Comment:
Shouldn't we change this line (`CLOUDWATCH_DEBUG`) instead of `LOGGING_TO_CLOUD_WATCH_ENABLED`?